pkg/scheduler/objects/allocation_ask.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/apache/yunikorn-core/pkg/events"
```

Comment:
maybe we can move the classes from `events` to `events/systems`? That can eliminate the reference conflicts.

Reply:
I don't want to introduce too much changes. I've been thinking about a better package name inside `objects`, but everything else is too long or not specific enough.